packages/@uppy/companion/src/server/provider/drive/index.js

```diff
@@ -112,18 +112,23 @@ class Drive extends Provider {
       .request()
   }
 
-  _getGsuiteFileMeta (id, token, mimeType, onDone) {
-    logger.info(`calling Gsuite file meta for ${id}`, 'provider.drive.export.meta')
-    return this.client
-      .query()
-      .head(`files/${id}/export`)
-      .qs({ supportsAllDrives: true, mimeType })
-      .auth(token)
-      .request(onDone)
+  _waitForFailedResponse (resp) {
+    return new Promise((resolve, reject) => {
+      let data = ''
+      resp.on('data', (chunk) => {
+        data += chunk
+      }).on('end', () => {
+        try {
+          resolve(JSON.parse(data.toString()))
+        } catch (error) {
+          reject(error)
+        }
+      })
```

Comment:
We need to handle `.on('error')` here too

Reply:
since the error caught [here](https://github.com/transloadit/uppy/pull/2223/files#diff-189fd85dd961eb623c8cce7945b3664cL148) is not conditional, my thought is that any error will still get bubbled up and handled there? 🤔

Reply:
oops yes you're right! `resp` already has an on('error') handler because of that. nevermind :)